data_api/api/models.py

```diff
@@ -237,15 +238,46 @@ def create_from_temba_list(cls, temba_list):
 
 class Group(Document, BaseUtil):
     org_id = StringField(required=True)
-    # created_on = DateTimeField()
```

Comment:
even though these were commented out, probably would be a good idea to track last_modified (and perhaps last_synced). could help with diagnosing sync issues in the future

Reply:
that makes sense. maybe we should add `last_synced` to all fields? I removed them because they are no longer provided by rapidpro.

Reply:
yup, might as well add for all synced objects
could probably sneak `first_synced` into `BaseUtil` but that seems less useful to have around compared to `last_synced`

Reply:
good call